packages/relay/src/lib/services/hapiService/hapiService.ts

```diff
@@ -99,12 +106,17 @@ export default class HAPIService {
     eventEmitter: EventEmitter,
   ) {
     dotenv.config({ path: findConfig('.env') || '' });
+    if (fs.existsSync(findConfig('.env') || '')) {
+      this.config = dotenv.parse(fs.readFileSync(findConfig('.env') || ''));
+    } else {
+      this.config = {};
+    }
```

Comment:
We don't need this additional field, please remove it.

The line above already loads those env variables into Node's `process.env`:
```typescript
dotenv.config({ path: findConfig('.env') || '' });
```

And those fallbacks that we are doing below in the added changes are essentially useless.

Reply:
@victor-yanev This is a workaround which properly handles the case where the environment defines an empty variable. The behavior of the `dotenv` module is to override the `.env` file even when the environment variable is an empty value or zero length string.

This change is needed to unblock the tests in CI and will be merged.

Reply:
@victor-yanev Additionally, the `OPERATOR_ID_MAIN` and `OPERATOR_KEY_MAIN` are truly not optional and must be supplied either via the `.env` or the actual environment variables.

The relay ceases to function if either of these variables are omitted. The current code does not properly handle or validate the absence of these required variables. This is a break fix for a problem in the code.

Reply:
Yes, that’s right but this `config` variable is still unneeded, because we are already loading the `.env` file into `process.env` through line 108.

Reply:
@victor-yanev I opened an [issue to handle environment variables properly in the relay.](https://github.com/hashgraph/hedera-json-rpc-relay/issues/3023)

Reply:
Sorry, I meant optional inputs in the workflow file, not optional overall in the relay.